src/index.ts

```diff
@@ -1,56 +1,298 @@
+      return prisma.business.update({
+        data: {
+          name: args.businessInput.name,
+          sustainabilityScore: args.businessInput.sustainabilityScore,
+          customerScore: args.businessInput.customerScore,
+          owner: args.businessInput.owner,
+          createdAt: args.businessInput.createdAt,
+          updatedAt: args.businessInput.updatedAt,
+
+          ProfileText: args.businessInput.ProfileText,
+          ProfilePicture: args.businessInput.ProfilePicture,
+          type: args.businessInput.type,
+          locationId: args.businessInput.locationId,
+          primaryLocation: args.businessInput.primaryLocation,
+          comments: args.businessInput.comments,
+          reviews: args.businessInput.reviews
+        },
+        where: {id: args.id}
+      })
+    },
+    updateLocation: (_parent, args: {id: number, locationInput: Prisma.LocationUpdateInput}) => {
+      return prisma.location.update({
+        data: {
+          address: args.locationInput.address,
+          createdAt: args.locationInput.createdAt,
+          updatedAt: args.locationInput.updatedAt,
+
+          longitude: args.locationInput.longitude,
+          latitude: args.locationInput.latitude,
+          businessName: args.locationInput.businessName,
+          Business: args.locationInput.Business,
+          Comment: args.locationInput.Comment
+        },
+        where: {id: args.id}
+      })
+    },
+    updateComment: (_parent, args: {id: number, commentInput: Prisma.CommentUpdateInput}) => {
+      return prisma.comment.update({
+        data: {
+          text: args.commentInput.text,
+          createdAt: args.commentInput.createdAt,
+          location: args.commentInput.location,
+          user: args.commentInput.user,
+
+          business: args.commentInput.business
+        },
+        where: {id: args.id}
+      })
+    },
+    updateReview: (_parent, args: {id: number, reviewInput: Prisma.UserReviewUpdateInput}) => {
+      return prisma.userReview.update({
+        data: {
+          createdAt: args.reviewInput.createdAt,
+          reputation: args.reviewInput.reputation,
+          ReviewData: args.reviewInput.ReviewData,
+          user: args.reviewInput.user,
+          Business: args.reviewInput.Business
+        },
+        where: {id: args.id}
+      })
+    },
+    deleteUser: (_parent, id: number) => {
+      return prisma.user.delete({
+        where: {id: id}
+      })
+    },
+    deleteBusiness: (_parent, id: number) => {
+      return prisma.business.delete({
+        where: {id: id}
+      })
+    },
+    deleteLocation: (_parent, id: number) => {
+      return prisma.location.delete({
+        where: {id: id}
+      })
+    },
+    deleteComment: (_parent, id: number) => {
+      return prisma.comment.delete({
+        where: {id: id}
+      })
+    },
+    deleteReview: (_parent, id: number) => {
+      return prisma.userReview.delete({
+        where: {id: id}
```

Comment:
Do these delete references to these objects in other entries in the DB?

Reply:
Will need to check.